services/wordpress/wordpress-platform.service.js

```diff
@@ -56,59 +22,38 @@ class WordpressPluginRequiresVersion extends BaseWordpressPlatform {
       {
         title: 'Wordpress Plugin: Required WP Version',
         namedParams: { slug: 'bbpress' },
-        staticPreview: this.render({ response: { requires: '4.8' } }),
+        staticPreview: this.render({ wordpressVersion: '4.8' }),
       },
     ]
   }
-}
 
-class WordpressPluginTestedVersion extends BaseWordpressPlatform {
-  static render({ version, color }) {
+  static get defaultBadgeData() {
+    return { label: 'wordpress' }
+  }
+
+  static render({ wordpressVersion }) {
     return {
-      message: `${addv(version)} tested`,
-      color,
+      message: addv(wordpressVersion),
```

Comment:
I don't personally have any objections to not including a `tested` suffix in the message, but any concerns about changing a message like this on an existing badge?

Reply:
Ack, this is a really gnarly diff, probably easier to inspect in split view. 😝 

The **tested** is still there in `WordpressPluginTestedVersion.render()` but the added code you're seeing is part of `WordpressPluginRequiresVersion.render()`.

Reply:
Oops! Thanks for that